mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -890,6 +911,43 @@ def __call__(self, results):
 
         return results
 
+    def __repr__(self):
+        """str: Return a string that describes the module."""
+        repr_str = self.__class__.__name__
+        repr_str += f'(num_points={self.num_points},'
+        repr_str += f' dist_metric={self.dist_metric})'
+
+        return repr_str
+
+
+@PIPELINES.register_module()
+class IndoorPointSample(PointSample):
+    """Indoor point sample.
+
+    Sampling data to a certain number.
+    NOTE: IndoorPointSample is deprecated in favor of PointSample
+
+    Args:
+        num_points (int): Number of points to be sampled.
+    """
+
+    def __init__(self, num_points):
```

Comment:
should use `__init__(self, *args, **kwargs):` and directly pass it to the super class.